components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -762,6 +762,42 @@ def _get_signature_from_request(request):
     return rv
 
 
+def _set_quantization_maps(quant_json, img):
```

Comment:
I think it would be better to have this functionality in the BlitzGateway so it can be tested independently of web and can be used by other Python clients.
Tests could be added under ```components/tools/OmeroWeb/test/integration/gatewaytest/test_rdefs.py``` although for "inverted" I added to ```components/tools/OmeroWeb/test/integration/test_rendering.py```

Reply:
Thanks for pointing me to the test locations.

the method you referred to above essentially just gathers the parameters from the json. most other clients using python would much rather want to call setQuantizationMap directly (without the json detour), right? there is also the wrapper on the image that the method above calls. Granted it is applicable to only single channels but still.